README.md

```diff
@@ -1 +1,237 @@
 
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <!--<img src="murple_logo.png" alt="logo" width="140"  height="auto" /> -->
```

Comment:
[REQUIRED] Kindly remove all commented-out codes. This makes your project look more professional

Reply:
Changes were made but, sorry were not committed. Thanks for the reminder.